tests/test_bear_deploy.py

```diff
@@ -31,7 +31,7 @@
     'config': {
      'NOInst': {
       'growth_commod': 'bears',
-      'growth_rate': 0.05,
+      'growth_rate': 0.1,
```

Comment:
why change this test in this way? (Sorry to be picky, but changes in the tests are usually a little bit of a no-no for me.) Adding tests is always fine, but altering or removing tests can allow changes to introduce backwards incompatibility, loss of functionality, etc.